esquery.js

```diff
@@ -128,25 +128,26 @@
                                 case 'literal': return '' + selector.value.value === '' + p;
                                 case 'type': return selector.value.value === typeof p;
                             }
+                            break;
```

Comment:
Are you sure all these fallthroughs were not deliberately omitted? (the eslint rule can be skipped if there is a `// Fallthrough` comment). Of course, if they were not deliberately omitted, it is better to add the `break`.

Reply:
Fairly sure -- did you look at the code? It seems very unlikely that the code for handling the `<=` operator is intentionally used as the fallback for the `=` and `!=`  operators when the selector has an unknown type? :thinking:

As I wrote in the commit message there should probably also be a `default` case in these switches that throws an error in case of an unknown selector.

Reply:
Cool, no I didn't look at it. Just asking to check it was considered. :)

Reply:
Cool, no I didn't look at it. Just asking to check it was considered. :) Thanks!

Reply:
I'd prefer putting a `throw` with a nice message here.

Reply:
Sure! Fixed in 246b39f